Automattic/woocommerce-payments
# Update: remove merchant onboarding E2E tests

Fixes #4009 

#### Changes proposed in this Pull Request


Removes the merchant onboarding E2E tests from the repo. The merchant onboarding tests are present in WooCommerce, maintaining them in 2 places like we do right now doesn’t seem ideal since we are just importing and calling the tests here.

Now that the importing flow is broken ([issue](https://github.com/woocommerce/woocommerce/issues/32604)), removing them from our repo along with the dependency to only keep WCPay related E2E tests here.






#### Testing instructions





* Go to Actions -> WCPay E2E tests -> Run workflow for this branch.
- Notice that the onboarding tests aren't included anymore

-------------------


**Post merge**
